index.js

```diff
@@ -73,16 +75,26 @@ function writeDescription (schema, options) {
         : typeMatch || schema.type
       }}`
   }
-  return `${indent(options)} *${description}
-${indent(options)} * @${options.objectTagName || 'typedef'}${type}${schema.title
+  let descriptionLine = ''
+  if (description || options.emptyDescriptionLineBreak !== false) {
```

Comment:
This one too, why not check for truthy? Since options are normally the exception they are usually "falsey" by default

Reply:
I tried to avoid changing the previous behavior since this were just stylistic options, so I set the default for these options to `true`.

If you care about preserving the old behavior, I could get the "options are exceptions" approach (with defaults as `false`) by renaming to `emptyDescriptionNoLineBreak` (and `propertiesNoLineBreak`). While this adds consistency as far as option defaulting behavior, it comes at the cost of requiring the user to process a double-negative.

Or if you don't care about the previous behavior, I can just switch to simpler code checks (and change the test expectations accordingly).

Let me know which you prefer.

Reply:
Thanks for preserving the behavior. I checked JSDoc again and don't see the extra line. This is again probably because I used some reference long time ago that had those. In this case, with all the new output changes and options which is going to be major version think it's justified to make these changes to better align with JSDoc. And with that, I don't know if users would still be interested in adding that extra line at all.

Reply:
As jsdoc is not a validator, it is pretty flexible with what one *can* do.

That being said, I'd tend to agree that there may be no need for `propertiesLineBreak` as I haven't seen that convention of adding a line break after `typdef` and before the first `property`.

However, as far as `emptyDescriptionLineBreak`, I think there are two reasons to at least keep this one as an option even if off by default:

1. The community has sought to require (as well as forbid) newlines after descriptions: https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-newline-after-description
2. Adding a line break saves a little time for people who know they will want to add their own description later.

Reply:
Alright, the `emptyDescriptionLineBreak` makes sense then (with the simplified truthy check).

Reply:
I've amended to drop `propertiesLineBreak` and changed `emptyDescriptionLineBreak` to be off by default (allowing the simplified truthy check), amending the other tests per the new default behavior.

Reply:
Reviewing it again and now I'm on the fence whether it there should be a line break or not as the default. (Because this affect how the option is named and as you mentioned, we want to avoid double negation) 

What's the trend in JSDoc nowadays?

Reply:
Maybe best explained with an example:
* No line as default: option can be called something like `addDescriptionLineBreak`
* With empty line as default: option can be called something like `removeDescriptionLineBreak`

Reply:
As far as the trend on line breaks after descriptions, I'm not sure I've discerned a particular pattern. Our `newline-after-description` rule defaults to "always" expecting a newline, but I've seen plenty of cases where there are no newlines. I personally like to avoid newlines as they involve slightly more work and visual size bloat, and tags on subsequent lines (esp. with IDE's coloring them distinctly) are enough to my taste to set them apart. But I'm fine either way.

Reply:
Ok, I'll lean towards aligning with JSDoc examples (no extra line) only because it's more likely to be referenced by a new user. 

I suggested `addDecriptionLineBreak` as I think the verb "add" communicates the intention and `emptyDescriptionLineBreak` _might_ be interpreted as "to make it empty" instead of "add an empty line". Thoughts?

Reply:
Makes sense, sounds good. Plan to change shortly.

Reply:
Amended. And rebased the docs PR as well.

Reply:
Btw, off topic, but since you are open to breaking changes (and it would lead to a simpler truthy check), I was wondering if you might be open to:

1. Making `autoDescribe` as `false` by default so that the `Represents an object` strings don't get added by default. This seems a bit redundant to me given that both the type and name are available within  `@typedef {type} name`, e.g., `@typedef {object} special` makes "Represents a special object" superfluous.
2. Making `capitalizeTitle` as `false` by default. I think it is safer not to modify the user's original data by default.
3. Make `descriptionPlaceholder` as `false` by default. I think it's cleaner to avoid adding placeholders even while it is a useful option for those who know they will continue to edit the jsdoc further after it is built.
4. Make `hyphenatedDescriptions` as `false` by default for the same reasons as no. 3.

Reply:
The auto description and capitalization was tailored to my use case back then (minimal schema --> documentation).

I'm good with turning these "modifications" off by default now. Examples in the readme can still illustrate that.

Reply:
Ok, cool--I assume you'll want that as a separate PR though (as those are preexisting settings)?

Reply:
Yeah, that makes sense

Reply:
So anything else you need on this PR?

Reply:
Nope, this is good